client/src/main/java/io/pravega/client/state/impl/RevisionImpl.java

```diff
@@ -44,4 +44,36 @@ public RevisionImpl asImpl() {
         return this;
     }
 
+    @Override
+    public String toString() {
```

Comment:
I actually overrode `toString()` on the Revision interface, so it has a javadoc indicating that it can be used this way, and forces the implementation to override the method.